.gitignore

```diff
@@ -1,8 +1,13 @@
 .coverage
+.idea
```

Comment:
is this file generated too?

Reply:
yea, generated by intellij ide usually. seems like a valid addition